app/managers.py

```diff
@@ -0,0 +1,50 @@
+import sqlite3
+
+from models import Actor
+
+
+class ActorManager:
+    def __init__(self) -> None:
+        self._connection = sqlite3.connect("cinema.sqlite")
+        self.table_name = "actors"
+
+    def all(self) -> list:
+        return [
+            Actor(*actor)
+            for actor
+            in self._connection.execute(f"SELECT * FROM {self.table_name}")
```

Comment:
Can you first make this execution into a separate variable, and after that make this list comprehension?